cdk/lib/recipes-backend.ts

```diff
@@ -229,6 +233,21 @@ export class RecipesBackend extends GuStack {
 			],
 		});
 
+		new Rule(this, 'ReindexConnection', {
+			eventBus,
+			description: `Connect recipe responder ${this.stage} to recipes-reindex`,
+			eventPattern: {
+				source: ['recipes-reindex'],
+			},
+			targets: [
+				new aws_events_targets.LambdaFunction(updaterLambda, {
+					deadLetterQueue: responderDLQ,
+					maxEventAge: Duration.minutes(30),
+					retryAttempts: 5,
```

Comment:
:+1: for including DLQ config.  Though it's worth remembering that this may effectively mix reindex and regular messages on the DLQ. Don't know if that would cause problems yet...